mod/settings.php

```diff
@@ -149,7 +149,7 @@ function settings_post(App $a)
 		check_form_security_token_redirectOnErr('/settings/oauth', 'settings_oauth');
 
 		$key = $_POST['remove'];
-		dba::delete('tokens', ['id' => $key, 'uid' => local_user()]);
+		dba::delete('tokens', ['id' => $key]);
```

Comment:
Don't worry, this has been brought back to live with the final commit of this PR.